scripts/database/releases.txt

```diff
@@ -0,0 +1,30 @@
+v4.0
+v4.0.1
+v4.1
+v4.2
+v4.2.1
+v4.2.2
+v4.2.3
+v4.2.4
+v4.3
+v4.3.1
+v4.4
+v4.5
+v4.5.1
+v4.6
+v4.6.1
+v4.6.2
+v4.7
+v4.7.1
+v4.8
+v4.8.1
+v4.8.2
+v4.8.3
+v4.8.4
+v4.8.5
+v4.8.6
+v4.9
+v4.9.1
+v4.9.2
+v4.9.3
+v4.9.4
```

Comment:
This long list of releases reminds me of doc/sphinx-guides/source/versions.rst and I wonder if there's an opportunity to consolidate these or have the rst file be build from the txt file?